lib/main_publiccloud.pm

```diff
@@ -114,6 +129,10 @@ sub load_latest_publiccloud_tests {
         if (get_var('PUBLIC_CLOUD_CONSOLE_TESTS')) {
             load_publiccloud_consoletests($args);
         }
+        elsif (check_var('PUBLIC_CLOUD_NVIDIA', 1)) {
+            die "ConfigError: The provider is not supported\n" unless (check_var('PUBLIC_CLOUD_PROVIDER', 'GCE'));
```

Comment:
no one can stop you from loading latest tests with SLE 12 SP5 :)  yes normally it should not happen but this is not how we decide if we need to check for something or not

Reply:
Please remind me. `load_latest_publiccloud_tests` vs `SLE 12 SP5 `, dont see the connection. My guess is that you ask me to include version check! am i right?

Reply:
yeah I am saying that check should be the same as for maintenance tests

Reply:
I am not sure if this is even a think here but i add the version check. *ready for review*